rules/manifest/applications-gecko-id.js

```diff
@@ -11,20 +11,22 @@ module.exports = class ApplicationsGeckoId extends Rule {
    * @param  {object} flags   Flags via the CLI parser.
    */
   constructor(...args) {
-    super(...args, __filename);
+    super(...args);
   }
 
-  validate() {
+  validate(severity="warn", ...options) {
     this.logger.verbose(this.name);
 
+    const log = this.logger[severity];
```

Comment:
Also this. If we eventually add custom config support, by passing a `severity` of "warn", we'd call `this.logger.warn()` and maintain an exit code of 0. If we pass a severity of "error", we'd call `this.logger.error()` which sets a `process.exitCode` of 1, so it'd potentially break the build or whatever. :+1: